### PR TITLE
New method: update (116 B)

### PR DESCRIPTION
**Pull request by BabkinAleksandr**
[Ramda update](http://ramdajs.com/docs/#update)

`slice`:
update - slice

`concat`:
update - concat

`for`:
update - for

### CODE REVIEW

File: TIMES.md [hunk 1]
```diff
@@ -51,6 +51,11 @@
 |             | diff  |  -0.04ms | +0.02ms |  +0.83ms |
 |             |       |  -35.38% |  +7.90% |  +29.81% |
 | ----------- | ----- | -------- | ------- | -------- |
+| update      | nano  |   0.41ms |  4.79ms |  64.93ms |
+|             | ramda |   0.13ms |  2.52ms |  42.18ms |
+|             | diff  |  +0.29ms | +2.27ms | +22.75ms |
+|             |       | +226.68% | +89.77% |  +53.94% |
+| ----------- | ----- | -------- | ------- | -------- |
```
Comment (Beraliv) [inline TIMES.md]: please, change `TIMES` for this approach

File: lib/update/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import _curry3 from '../_internal/_curry3'
+
+export default _curry3(function update(idx, value, list) {
+  var result = list.slice()
+  result[idx - 1] = value
```
Comment (Beraliv) [inline lib/update/index.js]: why `index - 1`?

Comment (BabkinAleksandr) [reply lib/update/index.js]: hmm, you're right. I thought that this method assume count from `1`, but I've mistaken

File: misc/cache/update.json [hunk 3]
```diff
@@ -0,0 +1 @@
...8359862014651297,0.08734694015234709],[4.290136689990759,2.517615950293839],[60.36459360983223,42.47421369004994]],[[0.3551221798360348,0.09728199005126953],[5.138966749981046,2.4195209401100874],[64.62077116992324,42.37339772000909]],[[0.41430597003549335,0.20326448000967504],[5.091953150033951,2.6070429399237036],[67.25955639995635,42.6075937801227]],[[0.44253095004707577,0.09234886985272169],[5.042823329940438,2.6033833500370385],[67.54201184015722,42.94348912011832]],[[0.41271155994385483,0.17138905003666877],[4.785555970072746,2.6304528503119946],[67.49635602019727,42.74596825025976]],[[0.39673642978072166,0.18360704988241194],[4.562935559935868,2.562992050200701],[65.02112306971102,41.44388154990971]],[[0.48040982011705635,0.14815956022590399],[4.478010229989886,2.4627621399611233],[60.75511858996004,41.86194291010499]],[[0.4580971998721361,0.09667419038712978],[5.1579531100764875,2.6539604198932647],[66.12452027000487,41.673235260099176]],[[0.45009478960186244,0.09349487990140914],[4.70534488003701,2.5599313700571655],[67.59217485018074,42.96074144005775]],[[0.32461491990834473,0.08706802986562252],[4.646456470228731,2.223476440384984],[62.53149614993483,40.70183436021209]]]
```
Comment (Beraliv) [inline misc/cache/update.json]: please remove the content of this file and rerun it again

Comment (BabkinAleksandr) [reply misc/cache/update.json]: [I've understood that](https://github.com/nanoutils/nanoutils/pull/63#discussion_r178030485)

**Pull request merged**